assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+                  "sanitary_dump_station:chemical_toilet=no"
+                ]
+              },
+              "then": {
+                "en": "You cannot dispose of chemical toilet waste here"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "access tag"
+          },
+          "question": {
+            "en": "Who can use this dump station?"
+          },
+          "freeform": {
+            "key": "access"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "access=network"
+                ]
+              },
+              "then": {
+                "en": "You need a network key/code to use this"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=customers"
+                ]
+              },
+              "then": {
+                "en": "You need to be a customer of camping/campersite to use this place"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "access=public"
+                ]
+              },
+              "then": {
+                "en": "Anyone can use this dump station"
+              },
+              "hideInAnswer": false
+            },
+            {
+              "if": {
+                "and": [
+                  "access=yes"
+                ]
+              },
+              "then": {
+                "en": "Anyone can use this dump station"
+              },
+              "hideInAnswer": true
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place is operated by {operator}"
+          },
+          "question": {
+            "en": "Who operates this place?"
+          },
+          "freeform": {
+            "key": "operator"
+          }
+        },
+        {
+          "render": {
+            "en": "This station is part of network {network}"
+          },
+          "question": {
+            "en": "What network is this place a part of?"
+          },
+          "freeform": {
+            "key": "network"
+          }
+        }
+      ],
+      "hideUnderlayingFeaturesMinPercentage": 0,
+      "icon": {
+        "render": "./assets/themes/campersites/sanitary_dump_station.svg"
+      },
+      "width": {
+        "render": "8"
+      },
+      "iconSize": {
+        "render": "40,40,center"
+      },
+      "color": {
+        "render": "#00f"
+      },
+      "presets": [
+        {
+          "tags": [
+            "amenity=sanitary_dump_station"
+          ],
+          "title": {
+            "en": "sanitary dump station"
+          },
+          "description": {
+            "en": "Add a new sanitary dump station. Please zoom in sufficiently for exact placement. Don't copy info from copyrighted content. Your data will be added straight to the global OpenStreetMap database!"
```

Comment:
Same remark as the other preset: the general warning message should be handled by MapComplete, not every theme individually

Reply:
done